### ISSUE TITLE
`LogEntry.prototype.withException` fails for exceptions raised by the VoiceID function.

### ISSUE THREAD

**Issue by intptr-t** [amazon-connect/amazon-connect-streams#394]
Now I'm using version 1.6.5 of amazon-connect-streams to evaluate VoiceID.

For the exception raised by calling the function of VoiceID, `typeof err.stack` is an object type, so an exception was raised in the exception.

In the implementation of the VoiceIdError class(function obj), it looks to me like the `error.stack = Error(message);` is not being assigned properly.
https://github.com/amazon-connect/amazon-connect-streams/blob/1.6.5/src/util.js#L659

The `stack` variable in the LoggedException object seems to be expecting a string.
https://github.com/amazon-connect/amazon-connect-streams/blob/1.6.5/src/log.js#L175

## Console log
```
[2021-04-05T08:19:55.633Z] [ERROR]: startVoiceIdSession failed
[2021-04-05T08:19:55.634Z] [ERROR]: evaluateSpeakerWithVoiceId failed
connect-streams.js:21370 Uncaught (in promise) TypeError: e.stack.split is not a function
    at new LoggedException (connect-streams.js:21370)
    at LogEntry.withException (connect-streams.js:21417)
    at index.js:245
```

## Excerpt of the code under evaluation

```js
/** @type {connect.VoiceConnection} type cast */
const voiceConnection = (baseConnection);
// @ts-ignore  index.d.ts is wrongly defined -> EvaluateSpeakerWithVoiceId(): Promise<any>;
voiceConnection.evaluateSpeakerWithVoiceId(true)
    .then((response) => {
        connect.getLog().info(`evaluateSpeakerWithVoiceId success`).withObject(response);
    })
    .catch((err) => {
        connect.getLog().error(`evaluateSpeakerWithVoiceId failed`).withException(err);
    });
```

**Comment by hartjo** [comment]
Did you manage to fix or implement the VoiceId with success?

i have a different error on this issue 

https://github.com/amazon-connect/amazon-connect-streams/issues/381

**Comment by intptr-t** [comment]
Thanks for the reply @hartjo 

#381 VoiceID response also fails though.
Apart from that, there is an error in the exception handling inside the Streams API.
That is what makes this issue.

```js
    .catch((err) => {
        try {
            // This process will throw an exception.
            connect.getLog().error(`evaluateSpeakerWithVoiceId failed`).withException(err);
        }
        catch(err) {
        }
    });
```

**Comment by hartjo** [comment]
I've been working on this for months. no luck as of today. contact flow working as expected but I'm stuck on this CPP enrollment.

**Comment by hartjo** [comment]
@intptr-t 

no update? lol

**Comment by intptr-t** [comment]
This issue has been resolved in [release 1.6.9](https://github.com/amazon-connect/amazon-connect-streams/commit/afdcb6d8d9e5f9628205c2ecf0317e9a92e9b3a5#diff-9c0ec1b0a889e599f3ff81590864dd9dc65684a86b41f1da75acc53fafed12e3R680).
